File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.cs
using System;
using System.Collections.Generic;
using System.Linq;
public class Solution {
    public string solution(string my_string, int[] indices) {
        string answer = "";
        List<int> arr = indices.ToList();
        arr.Sort();
        for(int i = 0 ; i < my_string.Length;i++)
        {
            bool isIn = false;
            for(int j = 0 ; j < arr.Count; j++)
            {
                if(arr[j] == i)
                {
                    isIn = true;
                    break;
                }
            }
            
            if(isIn == false)
            {
                answer += my_string[i];
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

string solution(string my_string, vector<int> indices) {
    for(int i = 0; i < indices.size(); i++)
        my_string[indices[i]] = ' ';
    
    my_string.erase(remove_if(my_string.begin(), my_string.end(), ::isspace), my_string.end());
    return my_string;
}